src/woo_publications/publications/tests/test_documents_admin.py

```diff
@@ -218,3 +220,119 @@ def test_publications_admin_delete(self):
         self.assertFalse(
             Document.objects.filter(identifier=document.identifier).exists()
         )
+
+    def test_document_admin_document_api_configuration(self):
+        publication = PublicationFactory.create()
+        service = ServiceFactory(
+            api_root="https://example.com/",
+            api_type=APITypes.drc,
+            oas="https://example.com/api/v1/oas",
+            header_key="Authorization",
+            header_value="Token 0cbccf9e-f9cd-4f9c-9516-7481e79989df",
+            auth_type=AuthTypes.api_key,
+        )
+
+        response = self.app.get(
+            reverse("admin:publications_document_add"),
+            user=self.user,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        with self.subTest("select box only shows document api options"):
+            zaak_service = ServiceFactory(
+                api_root="https://foo.com/",
+                api_type=APITypes.zrc,
+                oas="https://foo.com/api/v1/oas",
+                header_key="Authorization",
+                header_value="Token eef8d572-5fac-415c-9b8d-132b4b5eab19",
+                auth_type=AuthTypes.api_key,
+            )
```

Comment:
Creating this record *after* the `response` is retrieved will never show it, even if the `limit_choices_to` is mis-configured, simplify because at response render time there was only one service available. This needs to be moved to the start of the test were all the data is setup.